Tests/test_imagefont.py

```diff
@@ -164,7 +175,7 @@ def test_render_multiline_text(self):
             # to multiline_text()
             draw.text((0, 0), TEST_TEXT, fill=None, font=ttf, anchor=None,
                       spacing=4, align="left")
-            draw.text((0, 0), TEST_TEXT, None, ttf, None, 4, "left")
+            draw.text((0, 0), TEST_TEXT, None, ttf, None, None, 4, "left")
```

Comment:
This is an example of existing user code that is broken by changing the order of parameters. This change shouldn't be necessary.